feat: Azure get-well [COMP-806] (#1315)
seqeralabs/docs: platform-cloud/docs/studios/add-studio.md

   - **GPUs allocated**: Available only if the selected compute environment has GPU support enabled. For more information about GPUs on AWS, see [Amazon ECS task definitions for GPU workloads][aws-gpu]. The default allocation is 0 GPUs.
   - **Maximum memory allocated**: The default allocation is 8192 MiB of memory.

:::note

> In AWS Batch, Seqera creates two job queues and their respective compute environments: a head queue that runs the parent Nextflow process on a single On-Demand instance, and a worker queue that executes per-task processes dispatched by the head node, typically on Spot instances. Studios uses only the head queue and its compute environment. The worker queue is not used.

Reviewer: "and does not use the worker queue"

> In AWS Batch, Seqera creates two job queues and their respective compute environments: a head queue that runs the parent Nextflow process on a single On-Demand instance, and a worker queue that executes per-task processes dispatched by the head node, typically on Spot instances. Studios uses only the head queue and its compute environment. The worker queue is not used.

Author: ```suggestion
In AWS Batch, Seqera creates two job queues and their respective compute environments: a head queue that runs the parent Nextflow process on a single On-Demand instance, and a worker queue that executes per-task processes dispatched by the head node, typically on Spot instances. Studios uses only the head queue and its compute environment and does not use the worker queue.
```
